src/ray/raylet/placement_group_resource_manager.cc

```diff
@@ -22,6 +22,18 @@ namespace ray {
 
 namespace raylet {
 
+void PlacementGroupResourceManager::ReturnUnusedBundle(
+    const std::unordered_set<BundleID, pair_hash> &in_use_bundles) {
+  for (auto iter = bundle_spec_map_.begin(); iter != bundle_spec_map_.end();) {
+    if (0 == in_use_bundles.count(iter->first)) {
+      ReturnBundle(*iter->second);
+      bundle_spec_map_.erase(iter++);
```

Comment:
I think you shouldn't do `iter++` here. Afaik `erase` will move the iterator

Reply:
em.. I have tested, It seems not.